documentation/images/images-default.md

```diff
@@ -0,0 +1,58 @@
+⚠️ This documentation file isn't complete ⚠️
+
+It is only there to serve as an example of how we'd document each template
+
+---
+
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | images-default.stencil |
+| Invocation example | `swiftgen storyboard -t default …` |
+| Language | Swift 3 |
+| Author | Olivier Halligon |
+
+## When to use it
+
+This template is the default template used by SwiftGen when parsing images in Asset Catalogs.
+It generates Swift 3 code and is suitable for most needs.
+
+## Customization
```

Comment:
I think the whole explanation about "customize using --param" and that we load ENV variables, should be in the SwiftGen documentation itself. Here we should only document what is accepted (and in some cases "why"), and link to the swiftgen docs on how to use it.